include/deal.II/sundials/copy.h

```diff
@@ -0,0 +1,74 @@
+//-----------------------------------------------------------
+//
+//    Copyright (C) 2015 by the deal.II authors
```

Comment:
You may want to adjust the year.

Do we want to stick with the directory name `sundials/`? Or should we go with something more generic, say `odesolvers/`?

Reply:
sundials offers also nonlinear solvers and dae solvers (IDA is a dae solver, for example, not an ode solver...). I'd stick to sundials.

Reply:
i agree with @luca-heltai . Eventually we may have some more interfaces to other libs which contain multiple types of solvers so i would group headers based on the library name.